### PR TITLE
[MODULAR] [NO GBP] Adds alien/standard surgery tools + bonesetter to robo exofab

### PR DESCRIPTION
**Pull request by nikothedude**
## About The Pull Request

I just kinda forgot to do this in my last PR.

## How This Contributes To The Skyrat Roleplay Experience

There really isnt any reason they should be locked to adv. tools only.

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>

image

</details>

## Changelog

:cl:
add: Robo can now print standard and alien surgical tools
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/science_tools/medical_tool_designs.dm [hunk 1]
```diff
@@ -0,0 +1,165 @@
+// Standard tools
```
Comment (vinylspiders) [inline modular_skyrat/modules/science_tools/medical_tool_designs.dm]:
```suggestion
// Standard tool overrides
```

File: modular_skyrat/modules/science_tools/medical_tool_designs.dm [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+/datum/design/healthanalyzer_advanced
+	construction_time = 5 SECONDS
+
+/datum/design/healthanalyzer_advanced/New()
+	build_type |= MECHFAB
+	category += list(RND_CATEGORY_MECHFAB_EQUIPMENT + RND_SUBCATEGORY_MECHFAB_EQUIPMENT_MEDICAL)
+
+	return ..()
+
+// Alien tools
```
Comment (vinylspiders) [inline modular_skyrat/modules/science_tools/medical_tool_designs.dm]:
```suggestion
// Alien tool overrides
```

File: modular_skyrat/modules/science_tools/medical_tool_designs.dm [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+	construction_time = 5 SECONDS
+
+// it's fine to give them health analyzers, the choke for medibot production is the medkits
+/datum/design/healthanalyzer/New()
+	build_type |= MECHFAB
+	category += list(RND_CATEGORY_MECHFAB_EQUIPMENT + RND_SUBCATEGORY_MECHFAB_EQUIPMENT_MEDICAL)
+
+	return ..()
+
+// Advanced tools
```
Comment (vinylspiders) [inline modular_skyrat/modules/science_tools/medical_tool_designs.dm]:
```suggestion
// Advanced tool overrides
```

**Review by vinylspiders — CHANGES_REQUESTED** [review]
Just to make it clearer that we're looking at overrides here, if it wasn't obvious at first glance.

Usually they live in master_files so it's useful to mark them as such when you are putting them in a module.

**Comment by nikothedude** [comment]
gotcha

**Review by vinylspiders — APPROVED** [review]

**Pull request merged**